.gitignore

```diff
@@ -144,3 +144,11 @@ cython_debug/
 # Solar Car stuff
 google_maps_key.py
 ku-solar-car-b87af-firebase-adminsdk-ttwuy-0945c0ac44.json
+
+#Internal Testing
```

Comment:
If all of this test code is going to be ignored, maybe it should just be outside the repository? Or create a folder to contain it all instead of ignoring individual files.

Reply:
Cleaned up